filter.go

```diff
@@ -0,0 +1,7 @@
+package paperspace
+
+type Filter struct {
+	Where map[string]interface{} `json:"where"`
+	Limit int64                  `json:"limit"`
+	Order string                 `json:"order"`
```

Comment:
Should these be omitempty, ie you can have a where without a limit

Reply:
Yes, thank you for catching this.